src/internal/kopia/upload.go

```diff
@@ -720,18 +720,28 @@ func inflateCollectionTree(
 	// Temporary variable just to track the things that have been marked as
 	// changed while keeping a reference to their path.
 	changedPaths := []path.Path{}
+	// updatedLocations holds the location path of all moved collections. New
```

Comment:
Holds the previous path, or the end path?  Even after reading the comment, this is a little ambiguous.